README.md

```diff
@@ -1,75 +1,3 @@
-# Common Hosted Form Service [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE) [![img](https://img.shields.io/badge/Lifecycle-Stable-97ca00)](https://github.com/bcgov/repomountie/blob/master/doc/lifecycle-badges.md)
+# Common Hosted Form Service
 
-![Tests](https://github.com/bcgov/common-hosted-form-service/workflows/Tests/badge.svg)
-[![Maintainability](https://api.codeclimate.com/v1/badges/950b1d6c61567a1da227/maintainability)](https://codeclimate.com/github/bcgov/common-hosted-form-service/maintainability)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/950b1d6c61567a1da227/test_coverage)](https://codeclimate.com/github/bcgov/common-hosted-form-service/test_coverage)
-
-Create, edit and publish forms.
-
-## Directory Structure
-
-    .devcontainer              - Visual Studio Code devcontainer and local environment config
-    .github/                   - PR and Issue templates
-    .vscode/                   - Visual Studio Code's Launcher and Tasks
-    app/                       - Application Root
-    ├── frontend/              - Frontend Root
-    │   ├── src/               - Vue.js frontend web application
-    │   └── tests/             - Vue.js frontend web application tests
-    ├── src/                   - Node.js backend web application
-    │   ├── db/migrations      - data migration scripts
-    │   ├── docs/              - OpenAPI 3.0 Specification
-    │   └── forms/             - Models, Controllers, Routes for the forms
-    └── tests/                 - Node.js backend web application tests
-    components/                - Form.io Custom Components Library
-    openshift/                 - OpenShift-deployment and shared pipeline files
-    tests/                     - External test frameworks
-    ├── functional/            - Supporting functional tests
-    │   └── cypress/           - Cypress functional test suite
-    └── load/                  - Supporting load tests
-        └── load-test/         - Chefs API load test suite
-    CODE-OF-CONDUCT.md         - Code of Conduct
-    COMPLIANCE.yaml            - BCGov PIA/STRA compliance status
-    CONTRIBUTING.md            - Contributing Guidelines
-    LICENSE                    - License
-    SECURITY.md                - Security Policy and Reporting
-
```

Comment:
Are we removing all the info from the readme? Wouldn't this directory structure be valuable to have in the repo?

Reply:
@vijaivir yes I this will be helpful. @WalterMoar @jasonchung1871 what do you think?

Reply:
I think we want to keep the readme the way it was, but just update the links from the Wiki to Backstage. Maybe we need to have a big discussion about what belongs in the repo files and what belongs in Backstage (to me, anything about using the _repo_ should be in the repo, and anything about using the _deployed app_ should be in Backstage - but we should discuss!).